package.json

```diff
@@ -43,6 +43,7 @@
     "release": "^6.3.0",
     "rmfr": "^2.0.0",
     "rollup": "^2.45.2",
+    "rollup-plugin-inject-process-env": "^1.3.1",
```

Comment:
I only installed this dependency so I could inject environment variables build time.

Another strategy would be by using [`@rollup/plugin-replace`](https://www.npmjs.com/package/@rollup/plugin-replace)